BREAKING.md

```diff
@@ -28,6 +29,12 @@ The `getQuorum()` method on `IContainer` and the `quorum` member of `IContainerC
 ### `IChannel.summarize` split into sync and async
 `IChannel` now has two summarization methods instead of a single synchronous `summarize`. `getAttachSummary` is synchronous to prevent channel modifications during summarization, `summarize` is asynchronous.
 
+### `IFluidConfiguration` deprecated and `configuration` member removed from `ContainerRuntime`
+
+The `IFluidConfiguration` interface has been deprecated and will be removed in an upcoming release.  This will include removal of the `configuration` member of the `IContainerContext` at that time.  To inspect whether the document is in readonly state, you should instead query `container.readOnlyInfo.readonly`.
```

Comment:
can you add which packages the affected APIs are from?

Reply:
Added in latest